src/renderer/deviceMessages.ts

```diff
@@ -1,13 +1,34 @@
 import { BackendRemote } from './backend-com'
 
-export async function updateDeviceChats(accountId: number) {
+export async function updateDeviceChats(
+  accountId: number,
+  skipCurrentChangelog: boolean = false
+) {
+  const UpdateMessage = async (
+    label: string,
+    msg: Partial<Parameters<typeof BackendRemote.rpc.addDeviceMessage>[2]>
+  ) => {
+    if (skipCurrentChangelog) {
+      await BackendRemote.rpc.addDeviceMessage(accountId, label, null)
+    } else {
+      await BackendRemote.rpc.addDeviceMessage(accountId, label, {
+        text: null,
+        html: null,
+        viewtype: null,
+        file: null,
+        location: null,
+        overrideSenderName: null,
+        quotedMessageId: null,
+        ...msg,
+      })
+    }
+  }
+
   // const tx = window.static_translate
```

Comment:
Could be removed since dead code

Reply:
sure but on the other hand you might need it when translating the update message (which we currently don't do because it would require bringing the string into the android repo, waiting for translations and then pulling it again) - but iOS and android translate this string.

BTW: the `tx`, there is a script that searches the code for missing translation keys, it looks for `tx("key")`, that is the reason it is mapped everywhere to that variable name.